metrics/cli/docs/diagrams/deploy_command_sequence_diagram.puml

```diff
@@ -39,19 +44,30 @@ FirebaseService --> Deployer -- : sign in finished
 Deployer -> FirebaseService ++ : acceptTermsOfService()
 FirebaseService --> Bob : asks to accept terms of Firebase service
 Bob -> FirebaseService : accepts terms
-FirebaseService -> Deployer  -- : terms accepted
+FirebaseService --> Deployer  -- : terms accepted
 
 Deployer -> GCloudService ++ : createProject()
-
 GCloudService --> Bob : asks to enter the regionId
 Bob -> GCloudService : enters the regionId
-
 GCloudService --> Deployer -- : gcloud project has been created
 
-Deployer -> FirebaseService ++ : createWebApp(projectId)
+Deployer -> GCloudService ++ : configureProjectOrganization(projectId)
+GCloudService --> Bob : asks to configure the organization for the project
+
+opt
```

Comment:
Would it be great to make this optional step similar to the `Sentry` configuration? I believe it would be great to display two options: 
1. User configures the organization 
2. User skips the configuration
And only after these options the organization configuration finishes. It means the organization configuration should be finished in any case since it is started on the `asks to configure the organization for the project` step. 
The same suggestion for the `Oauth origins configuration` step.